### DISCUSSION TITLE
How are email notifications sent with Freeform?

### DISCUSSION THREAD

**Discussion by JulieVelghe** [solspace/craft-freeform#895] [Questions & Answers] [answered]
Hi, I was wondering, does freeform require the user to log in, in order to send any notification mails?
Formie needs that trigger, or a cronjob of sorts, to make sure the notifications are being send.
I wonder how freeform does this. Is the queue being run automatically with each submission?

**Comment by kjmartens** [comment, marked as answer]
Hi @JulieVelghe,

Freeform does not use the Craft Queue to send email notifications, so they happen instantly and do not need any sort of action on your part to have them triggered. 🙂

**Reply by nitriques** [reply]
Hi @kjmartens would it be possible to send emails using the queue ? Emails sometimes fails, SMTP servers can be slow and the queue would let us retry sending those emails. Thanks !

**Reply by kjmartens** [reply]
@nitriques we are actually working on an update for this right now. You can expect it in Freeform 5.4 (about 1 week or so). 🙂

**Reply by nitriques** [reply]
@kjmartens Sweet :D Thanks for the ultra quick follow up
